tests/data/lib-1/builder.json

```diff
@@ -1,10 +1,15 @@
 {
     "name": "lib-1",
+    "variables": {
+        "gradlew": "{source_dir}/gradlew"
+    },
     "build_steps": [
-        "echo \"build lib-1\""
+        "echo \"build lib-1\"",
+        "{source_dir}/gradlew"
```

Comment:
This should be `{gradlew}` right?

Reply:
I think this stemmed from two separate tests. I originally just had this and then realized it would be convenient to define a variable using `{source_dir}`. I can probably rip that first test out and remove this given that the other test now covers this I think.